openml-lightgbm/lightgbm-provider/src/test/java/com/feedzai/openml/provider/lightgbm/TestParameters.java

```diff
@@ -58,6 +78,8 @@ public static Map<String, String> getDefaultParameters() {
                 defaultValue = String.valueOf(((ChoiceFieldType) type).getDefaultValue());
             } else if (type instanceof BooleanFieldType) {
                 defaultValue = String.valueOf(((BooleanFieldType) type).isDefaultTrue());
+            } else if (type instanceof FreeTextFieldType) {
+                defaultValue = String.valueOf(((FreeTextFieldType) type).getDefaultValue());
```

Comment:
Not sure if that is the expected behaviour but if `.getDefaultValue()==null` then `String.valueOf(null) == "null"`. Juts pointing out not sure if that is the expected behavior.